extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelSupport.java

```diff
@@ -63,6 +65,29 @@ public static boolean isPublic(ClassInfo ci) {
         }
     }
 
+    public static boolean isPathFiltered(String path, Collection<String> excludePatterns, Collection<String> includePatterns) {
```

Comment:
I'd be for a cleanly non-negated name and semantics here, e.g. `isPathIncluded`

Reply:
done